### ISSUE TITLE
RFE: allow build with latest `setuptools` > 60

### ISSUE THREAD

**Issue by kloczek** [harlowja/fasteners#84]
Looks like 0.17.1 forces to use exact version of the setuptools.
```console
+ /usr/bin/python3 -sBm build -w --no-isolation
* Getting dependencies for wheel...
running egg_info
creating fasteners.egg-info
writing manifest file 'fasteners.egg-info/SOURCES.txt'
writing manifest file 'fasteners.egg-info/SOURCES.txt'

ERROR Missing dependencies:
        setuptools==57.0.0
```
I've tested below patch:
```patch
--- a/pyproject.toml~   2022-01-17 11:43:19.000000000 +0000
+++ b/pyproject.toml    2022-01-17 13:02:58.194551517 +0000
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools==57.0.0", "wheel"]
+requires = ["setuptools>=57.0.0", "wheel"]
 build-backend = "setuptools.build_meta"

```
Ans build with latest `setupttols` > 60 is fine
```console
+ /usr/bin/python3 -sBm build -w --no-isolation
* Getting dependencies for wheel...
running egg_info
creating fasteners.egg-info
writing manifest file 'fasteners.egg-info/SOURCES.txt'
writing manifest file 'fasteners.egg-info/SOURCES.txt'
* Building wheel...
running bdist_wheel
running build
running build_py
creating build
creating build/lib
creating build/lib/fasteners
copying fasteners/__init__.py -> build/lib/fasteners
copying fasteners/_utils.py -> build/lib/fasteners
copying fasteners/lock.py -> build/lib/fasteners
copying fasteners/process_lock.py -> build/lib/fasteners
copying fasteners/version.py -> build/lib/fasteners
creating build/lib/fasteners/pywin32
copying fasteners/pywin32/__init__.py -> build/lib/fasteners/pywin32
copying fasteners/pywin32/pywintypes.py -> build/lib/fasteners/pywin32
copying fasteners/pywin32/win32con.py -> build/lib/fasteners/pywin32
copying fasteners/pywin32/win32file.py -> build/lib/fasteners/pywin32
running egg_info
writing manifest file 'fasteners.egg-info/SOURCES.txt'
running install
running install_lib
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/fasteners
creating build/bdist.linux-x86_64/wheel/fasteners/pywin32
running install_egg_info
Copying fasteners.egg-info to build/bdist.linux-x86_64/wheel/fasteners-0.17.1-py3.8.egg-info
running install_scripts
Successfully built fasteners-0.17.1-py3-none-any.whl
```

**Comment by psarka** [comment]
Thanks a bunch, was experimenting with some stuff and left it by mistake :man_facepalming:

**Comment by psarka** [comment]
Released 0.17.2 with a fix, should be ok now.

**Comment by kloczek** [comment]
> Released 0.17.2 with a fix, should be ok now.

No git tag for that version. Probably content of your repo not pushed to github 😄

**Comment by kloczek** [comment]
Nevertheless .. thank you 😄
